.github/workflows/verify-code.yaml

```
@@ -1,63 +1,61 @@
 ---
-name: image-builder
-
+name: coding-checks
```

```suggestion
name: code-and-container-pipeline
```

can we use somethink like above or `operator-build-pipeline`? etc

---

done.